src/main/scala/com/ubirch/webui/models/keycloak/member/ClaimTransaction.scala

```diff
@@ -42,23 +42,27 @@ class ClaimTransaction(device: MemberResourceRepresentation, prefix: String, tag
 
     device.leaveGroup(unclaimedGroup)
 
+    //update password
+    import scala.collection.JavaConverters._
+    val newDevicePassword = user.getDefaultPasswordForDevice()
```

Comment:
getDefaultPassword... sounds like there is one password all devices use. Is this the case at this point? Or is it rather a method like getRandomPasswordForDevice()?

Reply:
Neither ;)
I renamed it getPasswordForDevice. More appropriate indeed!